### PR TITLE
Adds Bio

### PR DESCRIPTION
**Pull request by kenmaglio**

### CODE REVIEW

File: MAGDEVSTUDIOS.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+I DON'T ALWAYS COMMIT MY CODE, BUT WHEN I DO, I DO IT TO MASTER!
+
+#BREAKBUILDS
```
Comment (michaeltlombardi) [inline MAGDEVSTUDIOS.md]: Please don't break builds.

File: MAGDEVSTUDIOS.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+#IGETGIT
```
Comment (michaeltlombardi) [inline MAGDEVSTUDIOS.md]: Do you though? Does anyone?

**Comment by kenmaglio** [comment]
MERGE MY STUFFS

File: MAGDEVSTUDIOS.md (hunk 2 above)
Comment (kenmaglio) [reply MAGDEVSTUDIOS.md]: MIKE!!!!!! I DO

**Pull request merged**